cmd/migrate/postgres.go

```
@@ -182,3 +189,232 @@ func runPostgresMigration(dryRun bool, timeout int, allowIndexChanges bool) erro
+	rows, err := db.QueryContext(ctx, `
+		SELECT n.nspname, c.relname
+		FROM pg_index i
+		JOIN pg_class c ON c.oid = i.indexrelid
+		JOIN pg_namespace n ON n.oid = c.relnamespace
+		WHERE (NOT i.indisvalid OR NOT i.indisready)
+		  AND n.nspname NOT IN ('pg_catalog', 'information_schema')
+		ORDER BY 1, 2`)
+	if err != nil {
+		return fmt.Errorf("check for invalid indexes: %w", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)